lib/web/apps.go

```diff
@@ -215,20 +218,26 @@ func (h *Handler) createAppSession(w http.ResponseWriter, r *http.Request, p htt
 			ServerNamespace: apidefaults.Namespace,
 		},
 		SessionMetadata: apievents.SessionMetadata{
-			SessionID: identity.RouteToApp.SessionID,
-			WithMFA:   identity.MFAVerified,
+			SessionID:       identity.RouteToApp.SessionID,
+			WithMFA:         identity.MFAVerified,
+			IDPSessionIndex: ws.GetIDPSessionIndex(),
 		},
 		UserMetadata: userMetadata,
 		ConnectionMetadata: apievents.ConnectionMetadata{
 			RemoteAddr: r.RemoteAddr,
 		},
+		ClientMetadata: apievents.ClientMetadata{
+			UserAgent: r.UserAgent(),
+		},
 		PublicAddr: identity.RouteToApp.PublicAddr,
 		AppMetadata: apievents.AppMetadata{
 			AppURI:        result.App.GetURI(),
 			AppPublicAddr: result.App.GetPublicAddr(),
 			AppName:       result.App.GetName(),
+			AppLabels:     result.App.GetAllLabels(),
```

Comment:
Any reason we want to persist the app labels

Reply:
It gives us idemeum app id to make it easier to lookup.